README.md

```
@@ -19,15 +19,35 @@ https://api.kakomimasu.com
+※ `.env`ファイルが使用できます。\
+※`GITHUB_CLIENT_*`が未指定の場合、アカウントに関連する機能（BearerTokenを用いたAPI）は利用できません。ゲストモードによるゲーム参加は可能です。
 
 ## 開発者向け
 
+### DB準備
+
+開発にはPostgreSQLが動作するサーバが必要です。
+
+ローカル用のPostgresサーバーを別途用意しない場合は、Prismaによるローカル用のPostgresサーバを使用すると手軽に用意できます。
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)